x-pack/solutions/search/plugins/search_indices/public/components/index_management/index_management_app.tsx

```diff
@@ -0,0 +1,82 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { ChromeBreadcrumb, ScopedHistory } from '@kbn/core/public';
+import { IndexManagementPluginSetup } from '@kbn/index-management-shared-types';
+import { reactRouterNavigate } from '@kbn/kibana-react-plugin/public';
+import { KibanaPageTemplate } from '@kbn/shared-ux-page-kibana-template';
+import React, { useRef, useEffect, useCallback } from 'react';
+import { i18n } from '@kbn/i18n';
+import { useKibana } from '../../hooks/use_kibana';
+
+const breadcrumbText = i18n.translate('xpack.searchIndices.breadcrumb', {
+  defaultMessage: 'Content',
+});
+
+const breadcrumb = {
+  text: breadcrumbText,
+};
+
+export const IndexManagementApp: React.FC<{ indexManagement: IndexManagementPluginSetup }> = ({
```

Comment:
Could this have a name that places it within search solution? Otherwise its easy to read this as specifically relating to IndexManagement instead of something more particular.

Reply:
renamed component name to `SearchIndexManagementApp` in this [commit](https://github.com/elastic/kibana/pull/209599/commits/5cab9beff7b08381b557c6ef87ec2d7c60408279)